float64[16] camToWorld
